src/core/utils/checks/charge_neutrality.hpp

```diff
@@ -0,0 +1,33 @@
+#include <cmath>
+
+#include "PartCfg.hpp"
```

Comment:
No.

Reply:
?

Reply:
As we have iterated multiple times, please do not put dependencies on the core into utils.